recipes/libcurl/all/conanfile.py

```diff
@@ -378,29 +384,6 @@ def _patch_cmake(self):
         replace_in_file(self, cmakelists, "${NGHTTP2_INCLUDE_DIRS}", "${libnghttp2_INCLUDE_DIRS}")
         replace_in_file(self, cmakelists, "${NGHTTP2_LIBRARIES}", "libnghttp2::nghttp2")
 
-        # wolfssl
-        replace_in_file(self, cmakelists, "find_package(WolfSSL REQUIRED)", "find_package(wolfssl REQUIRED CONFIG)")
-        if Version(self.version) < "8.10.0":
-            replace_in_file(self, cmakelists, "${WolfSSL_LIBRARIES}", "${wolfssl_LIBRARIES}")
-            replace_in_file(self, cmakelists, "${WolfSSL_INCLUDE_DIRS}", "${wolfssl_INCLUDE_DIRS}")
-        else:
-            replace_in_file(self, cmakelists, "${WOLFSSL_LIBRARIES}", "${wolfssl_LIBRARIES}")
-            replace_in_file(self, cmakelists, "${WOLFSSL_INCLUDE_DIRS}", "${wolfssl_INCLUDE_DIRS}")
-
-        # INTERFACE_LIBRARY (generated by the cmake_find_package generator) targets doesn't have the LOCATION property.
-        # So skipp the LOCATION check in the CMakeLists.txt
-        replace_in_file(
```

Comment:
@perseoGI  - please revert the removal of this `replace_in_file`

This is needed for anyone using CMake older than 3.20 (or 3.22) - otherwise they will be faced with the very error this is trying to solve

```
CMake Error at CMakeLists.txt:2013 (get_target_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LOCATION" is not allowed.


CMake Warning at CMakeLists.txt:2015 (message):
  Bad lib in library list: OpenSSL::SSL


CMake Error at CMakeLists.txt:2013 (get_target_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LOCATION" is not allowed.


CMake Warning at CMakeLists.txt:2015 (message):
  Bad lib in library list: OpenSSL::Crypto


CMake Error at CMakeLists.txt:2013 (get_target_property):
  INTERFACE_LIBRARY targets may only have whitelisted properties.  The
  property "LOCATION" is not allowed.


CMake Warning at CMakeLists.txt:2015 (message):
  Bad lib in library list: ZLIB::ZLIB
```

This is caused by the Conan-generated targets not having the location property set, and older versions of CMake were not allowed to instrospect this property on interface librararies (newer versions are fine).

Please remember to be conservative with these cleanups - this definitely has a chance of causing issues for a (now small) number of users